src/sm.c

```diff
@@ -250,7 +251,7 @@ static SM_RESULT sm_close_begin_internal(SM_HANDLE sm)
         } while (1);
 
         /*Codes_SRS_SM_02_053: [ sm_close_begin shall set SM_CLOSE_BIT to 0. ]*/
-        (void)InterlockedAnd(&sm->state, ~(ULONG)SM_CLOSE_BIT);
+        (void)interlocked_and(&sm->state, ~(ULONG)SM_CLOSE_BIT);
```

Comment:
>ULONG [](start = 44, length = 5)

uint32_t

Reply:
Fixed.

---
In reply to: [501163534](https://github.com/Azure/azure-c-util/pull/66#discussion_r501163534) [](ancestors = 501163534)